src/components/form/__spec__.js

```diff
@@ -137,7 +138,7 @@ describe('Form', () => {
 
   describe('checkStickyFooter', () => {
     beforeEach(() => {
-      wrapper = shallow(<Form />);
+      wrapper = mount(<Form />);
```

Comment:
is this to do with the use of refs again? (`this._form`) - there may be a way we can get around not having to use `mount`.